faq/serverless-jobs.mdx

```diff
@@ -124,3 +142,27 @@ To add network restrictions on your resource, consult the [list of prefixes used
 ## Can I securely use sensitive information with Serverless Jobs?
 
 Yes, you can use sensitive data such as API secret keys, passwords, TLS/SSL certificates, or tokens. Serverless Jobs seamlessly integrates with [Secret Manager](/identity-and-access-management/secret-manager/), which allows you to securely reference sensitive information within your jobs. Refer to the [dedicated documentation](/serverless/jobs/how-to/reference-secret-in-job/) for more information.
+
+## How can I store data in my Serverless ressource?
+
+Serverless ressources are by default [stateless](/serverless/functions/concepts/#stateless), local storage is ephemeral.
+
+In different use cases like saving results of analysis, exporting data etc... it can be important to save data. Serverless ressources can be connected to ressources from the Scaleway ecosystem.
```

Comment:
```suggestion
In different use cases like saving results of analysis, exporting data etc... it can be important to save data. Serverless resources can be connected to resources from the Scaleway ecosystem.
```